test/tools/ossfuzz/yulProto_diff_ossfuzz.cpp

```diff
@@ -70,7 +70,7 @@ DEFINE_PROTO_FUZZER(Program const& _input)
 
 	// AssemblyStack entry point
 	AssemblyStack stack(
-		langutil::EVMVersion(),
+		langutil::EVMVersion(langutil::EVMVersion::istanbul()),
```

Comment:
The EVM version has been bumped up in the proto fuzzer harness only to make it possible to fuzz early. Should we can keep this until the default EVM target of Solidity is bumped up (for the 0.5.14 release)?

Reply:
That's the purpose of the fuzzer, find bugs before we release :)